src/FrontendIntegration/FrontendFilter.php

```diff
@@ -427,8 +428,11 @@ protected function getFilters()
             }
         }
 
+        $values         = \array_merge($all->getSlugParameters(), $all->getGetParameters());
+        $baseParameters = $this->getBaseParameters($values);
+
         $arrWidgets = $filterSetting->getParameterFilterWidgets(
-            \array_merge($all->getSlugParameters(), $all->getGetParameters()),
+            \array_merge($baseParameters, $values),
```

Comment:
This seems wrong to me.
Parameters seem to be in the wrong order IMO, as the method `getBaseParameters()` takes care of handling overriding.
This way, URL parameters always override static ones, even if "Allow get" is not set.

Reply:
you suggested the order like this... hmm?

Reply:
Then I was wrong.